### PR TITLE
Update configuration page in core docs  - Closes #1560 

### LINKED ISSUE #1560: Update `Configuration`
### Description

- Update `Configuration` to use the interface of Lisk Core v6

No big changes are expected, but check with lisk-core team to ensure all the steps are correct

- Remove --enable-forger-plugin --enable-http-api-plugin
  - http-api plugin is removed
  - name of forger plugin cannot be changed, so removing from example

#### Affected branch / URL
https://lisk.com/documentation/lisk-core/management/configuration.html

### PR DESCRIPTION
**Pull request by chris529**
Closes #1560

### CODE REVIEW

File: modules/ROOT/pages/management/configuration.adoc [hunk 1]
```diff
@@ -0,0 +1,159 @@
+= Configuration
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/management/configuration.adoc:1]: Please add a navigation link to the page in the following file:
https://github.com/LiskHQ/lisk-docs/blob/1560-update-configuration-page-core/modules/ROOT/nav.adoc

File: modules/ROOT/pages/management/configuration.adoc [hunk 2]
```diff
@@ -59,24 +59,24 @@ $ lisk-core start --config=/home/lisk/custom-config.json --overwrite-config
 
 NOTE: Remember to xref:{url_pm2_restart}[restart] the node for the new config changes to take effect.
 
-== Enabling plugins
+// == Enabling plugins
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/management/configuration.adoc]: This section is still relevant and correct, so just remove the HTTP API Plugin from the description.

File: modules/ROOT/pages/management/configuration.adoc [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+:url_config_start: reference/cli.adoc#start
+:url_config_structure: reference/cli.adoc#structure
+:url_docker: management/docker.adoc
+:url_enable_pom: management/enabling-misbehavior-report.adoc
+:url_management_forging: management/forging.adoc
+:url_pm2_restart: management/pm2.adoc#refreshing-restarting-lisk-core
+:url_source: management/source.adoc
+// :url_sdk_plugins: lisk-sdk::plugins/index.adoc
+
+NOTE: A complete command reference with all available options for starting Lisk Core can be found in the xref:{url_config_start}[Lisk Core CLI].
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/management/configuration.adoc]:
```suggestion
//NOTE: A complete command reference with all available options for starting Lisk Core can be found in the xref:{url_config_start}[Lisk Core CLI].
NOTE: A complete command reference with all available options for starting Lisk Core can be found in Lisk Core CLI's reference page.
```

File: modules/ROOT/pages/management/configuration.adoc [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+NOTE: Remember to xref:{url_pm2_restart}[restart] the node for the new config changes to take effect.
+
+// == Enabling plugins
+
+// Plugins are not enabled by default.
+// Use the dedicated flags to enable them on application start:
+
+// [source,bash]
+// ----
+// $ lisk-core start --enable-forger-plugin
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/management/configuration.adoc]: This section about `Enabling Plugin` is still relevant. The change suggested in the issue is to remove `--enable-forger-plugin --enable-http-api-plugin` and use a different plugin name, for example, you can use  the following snippet:
```suggestion
$ lisk-core start --enable-report-misbehavior-plugin
```

File: modules/ROOT/pages/management/configuration.adoc [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+----
+
+NOTE: If you intend to connect the node to the Mainnet, change `testnet` to `mainnet` in the snippet above.
+
+[source,bash]
+----
+$ lisk-core start --config=/home/lisk/custom-config.json --overwrite-config
+----
+
+NOTE: Remember to xref:{url_pm2_restart}[restart] the node for the new config changes to take effect.
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/management/configuration.adoc]:
```suggestion
//NOTE: Remember to xref:{url_pm2_restart}[restart] the node for the new config changes to take effect.
NOTE: Remember to restart the node for the new config changes to take effect.
```

File: modules/ROOT/pages/management/configuration.adoc [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+:url_config_clo: reference/cli.adoc#clo
+:url_config_start: reference/cli.adoc#start
+:url_config_structure: reference/cli.adoc#structure
+:url_docker: management/docker.adoc
+:url_enable_pom: management/enabling-misbehavior-report.adoc
+:url_management_forging: management/forging.adoc
+:url_pm2_restart: management/pm2.adoc#refreshing-restarting-lisk-core
+:url_source: management/source.adoc
+// :url_sdk_plugins: lisk-sdk::plugins/index.adoc
+
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/management/configuration.adoc]:
```suggestion
//TODO: Update the commented out hyperlinks once the pages are available.
```

File: modules/ROOT/pages/management/configuration.adoc [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+[source,bash]
+----
+$ lisk-core start --network=NETWORK <1>
+----
+
+<1> `NETWORK` may be `mainnet`, `testnet` or `devnet`.
+
+== Using a custom config file
+
+Lisk Core uses the default config for connecting to the Mainnet by default.
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/management/configuration.adoc]:
```suggestion
Lisk Core uses the default config for connecting to the mainnet by default.
```

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]
A few suggestions :)

File: modules/ROOT/pages/management/configuration.adoc (hunk 1 above)
Comment (chris529) [reply modules/ROOT/pages/management/configuration.adoc:1]: Good point! Thanks Talha, it is now done 👍

File: modules/ROOT/pages/management/configuration.adoc (hunk 4 above)
Comment (chris529) [reply modules/ROOT/pages/management/configuration.adoc]: OK cool..👍

File: modules/ROOT/pages/management/configuration.adoc [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+----
+
+Alternatively, enable the plugins directly in the config file, see <<using-a-custom-config-file>>.
+
+[TIP]
+====
+For more detailed descriptions of the available plugins, go to the xref:{url_sdk_plugins}[].
+
+// See also xref:{url_enable_pom}[] and xref:{url_config_start}[Lisk Core CLI: start]
+See also Report Misbehaviour plugin and the `Lisk Core CLI: start` command.
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/management/configuration.adoc]:
```suggestion
See also the **Report Misbehaviour** plugin and the `Lisk Core CLI: start` command.
```

File: modules/ROOT/pages/management/configuration.adoc [hunk 9]
```diff
@@ -0,0 +1,162 @@
...
+<1> `NETWORK` may be `mainnet`, `testnet` or `devnet`.
+
+== Using a custom config file
+
+Lisk Core uses the default config for connecting to the mainnet by default.
+
+However, it is possible to use Lisk Core with a custom config, by specifying the path to the config file with the `--config` or `-c` flag.
+
+The custom config file needs to follow the JSON config structure of Lisk Core.
+A straightforward and easy method is to copy one of the network-specific default configs of Lisk Core and adjust it to suit your requirements.
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/management/configuration.adoc]:
```suggestion
A straightforward method is to copy one of the network-specific default configs of Lisk Core and adjust it to suit your requirements.
```

File: modules/ROOT/pages/management/configuration.adoc [hunk 10]
```diff
@@ -0,0 +1,162 @@
...
+$ lisk-core start --config=/home/lisk/custom-config.json --overwrite-config
+----
+
+//NOTE: Remember to xref:{url_pm2_restart}[restart] the node for the new config changes to take effect.
+NOTE: Remember to restart the node for the new config changes to take effect.
+
+== Enabling plugins
+
+Plugins are not enabled by default.
+Use the dedicated flags to enable them on application start:
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/management/configuration.adoc]:
```suggestion
Use the dedicated flags to enable them on the application start:
```

File: modules/ROOT/pages/management/configuration.adoc [hunk 11]
```diff
@@ -0,0 +1,162 @@
...
+A straightforward and easy method is to copy one of the network-specific default configs of Lisk Core and adjust it to suit your requirements.
+
+./home/lisk/
+[source,bash]
+----
+cp .lisk/lisk-core/config/testnet/config.json custom-config.json
+vim custom-config.json
+----
+
+NOTE: If you intend to connect the node to the mainnet, change `testnet` to `mainnet` in the snippet above.
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/management/configuration.adoc]:
```suggestion
NOTE: If you intend to connect the node to the mainnet, change the `testnet` to the `mainnet` in the snippet above.
```

File: modules/ROOT/pages/management/configuration.adoc [hunk 12]
```diff
@@ -0,0 +1,162 @@
...
+
+This will use the default config file for the specific network when starting Lisk Core.
+The default configurations are located in the `config` folder of Lisk Core.
+
+[source,bash]
+----
+$ lisk-core start --network=NETWORK <1>
+----
+
+<1> `NETWORK` may be `mainnet`, `testnet` or `devnet`.
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/management/configuration.adoc]:
```suggestion
<1> `NETWORK` may be `mainnet`, `testnet`, or `devnet`.
```

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]
Looking better! 👍🏼

**Review by TalhaMaliktz — DISMISSED** [review]
LGTM 👍🏼

**Review by nagdahimanshu — APPROVED** [review]
Looks good 👍

**Pull request merged**